doc/manual/en/config/widgets/unknown/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _unknown:
+
+The Unknown widget
+==================
+
+.. api-doc:: Unknown
+
+Description
+-----------
+
+.. ###START-WIDGET-DESCRIPTION### Please do not change the following content. Changes will be overwritten
+
+Fallback widget shown when an unknown widget is defined in the configuration.
+You must not use this one directly.
```

Comment:
Can we suppress the docu generation for this widget?

It's just an error fallback.